update_fingerprints.py

```diff
@@ -133,8 +149,11 @@ def main(turns, repetitions):
 fp.plot()
 ```
     """.format(version, turns, repetitions)
+    try:
```

Comment:
Thank you for the tip @marcharper. Sorry for the delay I will take care of it now.